cmd/kaspawallet/libkaspawallet/bip32/extended_key.go

```diff
@@ -79,10 +79,6 @@ func (extKey *ExtendedKey) DeriveFromPath(pathString string) (*ExtendedKey, erro
 }
 
 func (extKey *ExtendedKey) path(path *path) (*ExtendedKey, error) {
-	if path.isPrivate && !extKey.IsPrivate() {
```

Comment:
Since the big M notation for paths (M/1/2/3) is almost never used, and it's very annoying to change the path to start with "M" and not "m" every time you deal with extended public keys, I decided to remove this restriction